src/Common/Description/GenericDescription.tsx

```diff
@@ -1,327 +1,238 @@
-    DATE_TIME_FORMATS,
-    deepEqual,
-    showError,
-    toastAccessDenied,
-} from '..'
+import { DESCRIPTION_EMPTY_ERROR_MSG, DESCRIPTION_UNSAVED_CHANGES_MSG, deepEqual, showError } from '..'
 import './genericDescription.scss'
 import { ReactComponent as Edit } from '../../Assets/Icon/ic-pencil.svg'
-import { GenericDescriptionProps } from './types'
-import { DEFAULT_MARKDOWN_EDITOR_PREVIEW_MESSAGE, MARKDOWN_EDITOR_COMMANDS } from '../Markdown/constant'
-import { MDEditorSelectedTabType, MD_EDITOR_TAB } from './constant'
-import { patchApplicationNote, patchClusterNote } from './service'
+import { GenericDescriptionProps, MDEditorSelectedTabType, MD_EDITOR_TAB } from './types'
+import {
+    DEFAULT_MARKDOWN_EDITOR_PREVIEW_MESSAGE,
+    MARKDOWN_EDITOR_COMMANDS,
+    MARKDOWN_EDITOR_COMMAND_TITLE,
+    MARKDOWN_EDITOR_COMMAND_ICON_TIPPY_CONTENT,
+} from '../Markdown/constant'
+import { ReactComponent as HeaderIcon } from '../../Assets/Icon/ic-header.svg'
+import { ReactComponent as BoldIcon } from '../../Assets/Icon/ic-bold.svg'
+import { ReactComponent as ItalicIcon } from '../../Assets/Icon/ic-italic.svg'
+import { ReactComponent as LinkIcon } from '../../Assets/Icon/ic-link.svg'
+import { ReactComponent as StrikethroughIcon } from '../../Assets/Icon/ic-strikethrough.svg'
+import { ReactComponent as CodeIcon } from '../../Assets/Icon/ic-code.svg'
+import { ReactComponent as QuoteIcon } from '../../Assets/Icon/ic-quote.svg'
+import { ReactComponent as ImageIcon } from '../../Assets/Icon/ic-image.svg'
+import { ReactComponent as OrderedListIcon } from '../../Assets/Icon/ic-ordered-list.svg'
+import { ReactComponent as UnorderedListIcon } from '../../Assets/Icon/ic-unordered-list.svg'
+import { ReactComponent as CheckedListIcon } from '../../Assets/Icon/ic-checked-list.svg'
 
-// Import images from dashboard
-// import { ReactComponent as HeaderIcon } from '../../../assets/icons/mdeditor/ic-header.svg'
-// import { ReactComponent as BoldIcon } from '../../../assets/icons/mdeditor/ic-bold.svg'
-// import { ReactComponent as ItalicIcon } from '../../../assets/icons/mdeditor/ic-italic.svg'
-// import { ReactComponent as LinkIcon } from '../../../assets/icons/mdeditor/ic-link.svg'
-// import { ReactComponent as StrikethroughIcon } from '../../../assets/icons/mdeditor/ic-strikethrough.svg'
-// import { ReactComponent as CodeIcon } from '../../../assets/icons/mdeditor/ic-code.svg'
-// import { ReactComponent as QuoteIcon } from '../../../assets/icons/mdeditor/ic-quote.svg'
-// import { ReactComponent as ImageIcon } from '../../../assets/icons/mdeditor/ic-image.svg'
-// import { ReactComponent as OrderedListIcon } from '../../../assets/icons/mdeditor/ic-ordered-list.svg'
-// import { ReactComponent as UnorderedListIcon } from '../../../assets/icons/mdeditor/ic-unordered-list.svg'
-// import { ReactComponent as CheckedListIcon } from '../../../assets/icons/mdeditor/ic-checked-list.svg'
+import Moment12HourFormat from './constant'
```

Comment:
Please revert the default export, reason being things like utils, constants, index files, etc are bound to export multiple things